packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-page-wrapper/entity-page-header.tsx

```diff
@@ -24,7 +24,7 @@ export const EntityPageHeader = ({ entity }: { entity: EntityResponse }) => {
         crumbs={[
           {
             title: "Entities",
-            href: `${accountSlug}/entities`,
+            href: `/${accountSlug}/entities`,
```

Comment:
Bonus: fixes top-left link in the Entity editor

<img width="276" alt="Screenshot 2022-10-21 at 13 29 52" src="https://user-images.githubusercontent.com/608862/197196070-1e1a6896-8948-455f-9b0f-5dc8ba178e9f.png">